test/e2e/network_segmentation_default_network_annotation.go

```diff
@@ -96,6 +98,27 @@ var _ = Describe("Network Segmentation: Default network multus annotation", func
 		Expect(netStatus[0].IPs).To(ConsistOf(exposedIPs), "Should have the IPs specified in the default network annotation")
 		Expect(strings.ToLower(netStatus[0].Mac)).To(Equal(strings.ToLower(tc.mac)), "Should have the MAC specified in the default network annotation")
 
+		By("Create second pod with default network annotation requesting the same MAC request")
+		pod2 := e2epod.NewAgnhostPod(f.Namespace.Name, "pod-mac-conflict", nil, nil, nil)
+		pod2.Annotations = map[string]string{"v1.multus-cni.io/default-network": fmt.Sprintf(`[{"name":"default", "namespace":"ovn-kubernetes", "mac":%q}]`, tc.mac)}
+		pod2.Spec.Containers[0].Command = []string{"sleep", "infinity"}
+		pod2, err = f.ClientSet.CoreV1().Pods(f.Namespace.Name).Create(context.Background(), pod2, metav1.CreateOptions{})
+		Expect(err).NotTo(HaveOccurred())
+
+		By("Asserting second pod has event attached reflecting MAC conflict error")
+		expectedErr := fmt.Sprintf("MAC address conflict detected: %s already allocated in network attachment %s/%s",
+			strings.ToLower(tc.mac), f.Namespace.Name, udn.Name)
+		Eventually(func(g Gomega) []corev1.Event {
+			events, err := f.ClientSet.CoreV1().Events(pod2.Namespace).Search(scheme.Scheme, pod2)
+			g.Expect(err).NotTo(HaveOccurred())
+			return events.Items
+		}).WithTimeout(time.Minute * 1).WithPolling(time.Second * 3).Should(ContainElement(SatisfyAll(
+			HaveField("Type", "Warning"),
+			HaveField("Reason", "ErrorAllocatingPod"),
+			HaveField("Message", ContainSubstring(expectedErr)),
+		)))
+		Expect(pod2.Status.Phase).ToNot(Equal(corev1.PodSucceeded))
```

Comment:
shouldn't you add new entries to this table ?

Reply:
I dont think so, the entry pass MAC/IPs for this test one MAC address its enough.

IMO this test shouldn't be a table in the first place because there is only one etnry.